hvplot/converter.py

```diff
@@ -118,7 +118,7 @@ class HoloViewsConverter(object):
     rot: number
         Rotates the axis ticks along the x-axis by the specified
         number of degrees.
-    shared_axes (default=False): boolean
+    shared_axes (default=True): boolean
```

Comment:
While I agree with this change, doesn't it need discussion?  Seems like a change in behavior that should be called out.

Reply:
There was a discrepancy between the docstring and the implementation. It `shared_axes` **was** and **is** `True` by default